RefrigeratorSaraFarkash/RefrigeratorSaraFarkash/Program.cs

```diff
@@ -0,0 +1,292 @@
+﻿// See https://aka.ms/new-console-template for more information
+
+using RefrigeratorSaraFarkash;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+
+namespace exeRefreg
+{
+    class Program
+    {
+        public static List<CRefrigerator> SortRefrigerator(List<CRefrigerator> refrigerators)
```

Comment:
you dont need this function

Reply:
I have no problem deleting it
But it is write in the section:
Sorting: C